packages/ocs/storage-pool/StoragePoolListPage.tsx

```diff
@@ -258,21 +276,18 @@ const RowRenderer: React.FC<RowProps<StoragePool, CustomData>> = ({
     listPagePath,
   }: CustomData = rowData;
 
-  const { name } = obj.metadata;
   const poolType = obj.type;
   const hideItems =
     poolType === POOL_TYPE.FILESYSTEM
       ? [ModalKeys.EDIT_LABELS, ModalKeys.EDIT_ANN]
       : [];
   const replica = obj.spec?.replicated?.size;
   const mirroringStatus: boolean = obj.spec?.mirroring?.enabled;
-  const mirroringImageHealth: string = mirroringStatus
-    ? obj.status?.mirroringStatus?.summary?.image_health
-    : '-';
-  const lastChecked: string = obj.status?.mirroringStatus?.lastChecked;
-  const formatedDateTime = lastChecked
-    ? twelveHoursdateTimeNoYear.format(new Date(lastChecked))
-    : '-';
+  const imageHealth =
+    !(!!mirroringImageHealthError && mirroringImageHealthLoading) &&
```

Comment:
`!(!!mirroringImageHealthError && mirroringImageHealthLoading)` will translate to `!mirroringImageHealthError || !mirroringImageHealthLoading`...
is that what was expected ??
```suggestion
    (!mirroringImageHealthError && !mirroringImageHealthLoading) &&
```